From 58c9d1086e21619d3db696c8df38ebad648bef88 Mon Sep 17 00:00:00 2001
From: qianbi1999 <zhoujian157@huawei.com>
Date: Wed, 26 Nov 2025 10:10:50 +0800
Subject: [PATCH 2/3] Update paths in retool_sft_preprocess.py

- Modified the `tools_config_file` path to correctly locate the tools configuration.
- Updated the SFT training dataset save path to the new desired location.
---
 recipe/retool/retool_sft_preprocess.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/recipe/retool/retool_sft_preprocess.py b/recipe/retool/retool_sft_preprocess.py
index 57d53c91..d2e22a55 100644
--- a/recipe/retool/retool_sft_preprocess.py
+++ b/recipe/retool/retool_sft_preprocess.py
@@ -125,12 +125,12 @@ def process(row: dict, *, tools: str):
 
 
 if __name__ == "__main__":
-    tools_config_file = "recipe/retool/sandbox_fusion_tool_config.yaml"
+    tools_config_file = "sandbox_fusion_tool_config.yaml"
     tools_config = OmegaConf.load(tools_config_file)
     tool_schema = OmegaConf.to_container(tools_config["tools"][0]["tool_schema"])
     tools = json.dumps([tool_schema])
 
     data = datasets.load_dataset("JoeYing/ReTool-SFT")["train"]
     data = data.map(process, fn_kwargs={"tools": tools})
-    save_path = os.path.expanduser("~/ReTool-SFT/data/train-00000-of-00001.parquet")
+    save_path = os.path.expanduser("./ReTool-SFT/data/train-00000-of-00001.parquet")
     data.to_parquet(save_path)
-- 
2.50.1.windows.1

